src/Idris/Core/ProofState.hs

```diff
@@ -576,7 +564,7 @@ complete_fill ctxt env (Bind x (Guess ty val) sc) =
        ns <- unify' ctxt env (valty, Just $ SourceTerm val')
                              (ty, Just ExpectedType)
        ps <- get
-       let (uh, uns) = unified ps
+--        let (uh, uns) = unified ps
```

Comment:
Is it possible to just remove the unused code instead of commenting it out?

Reply:
I know the other code is also commented, but I am just wondering whether there is any use in having commented out code or it just clutters the source files.
I will let it be up to your discretion

Reply:
I'll kill this.  I didn't want to assume the Powers That Be were amenable to deleting commented-out code (yet, maybe a separate PR?)